src/Framework/NativeMethods.cs

```diff
@@ -1478,15 +1478,13 @@ internal static void VerifyThrowWin32Result(int result)
     internal static extern bool GetFileAttributesEx(String name, int fileInfoLevel, ref WIN32_FILE_ATTRIBUTE_DATA lpFileInformation);
 
     [DllImport(kernel32Dll, SetLastError = true, CharSet = CharSet.Unicode)]
-    private static extern uint SearchPath
-    (
+    private static extern uint SearchPath(
```

Comment:
This method isn't used anywhere and is private. Can we remove it?

Reply:
Dead code can be removed

Reply:
I think the question was whether anyone calls SearchPath via reflection. I have no idea, personally, and I'd be willing to remove it, but I know I'm on the risky edge for most changes.

Reply:
It would be really strange to call it via reflection instead of just implementing their own p/invoke. It would make more sense to write a public method that exposes and then they can use it directly and avoid Reflection.

I can't find any usages of it via a "FindAll"

Reply:
> just implementing their own p/invoke

Fair enough. I'd been thinking this was a normal method rather than external.

Reply:
Thanks for reviewing, I really appreciate the input! My coding career has taught me I am forever fallible :)